src/pages/Team/components/List/style.css

```diff
@@ -147,3 +152,22 @@ p {
   }
 }
 
+.divPag {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  text-align: right;
+  width: 100%;
+  padding: 14px 4px;
+  margin: 0 auto;
+}
+
+.displayFlex {
```

Comment:
it is the same  situation as above, but here we use CSS property as class name, but this class has inside more than only display: flex

Reply:
corrected